README.md

```diff
@@ -67,6 +67,7 @@ If you want to support IE9, you will need to use a classList pollyfill, like Eli
 * __forcePlainText__: Forces pasting as plain text. Default: true
 * __placeholder__: Defines the default placeholder for empty contenteditables when __disablePlaceholders__ is not set to true. You can overwrite it by setting a data-placeholder attribute on your elements. Default: 'Type your text'
 * __secondHeader__: HTML tag to be used as second header. Default: h4
+* __standardizeSelectionStart__: Standardizes how the beginning of a range is decided between browsers whenever the selected text is analyzed for updating toolbar buttons status
```

Comment:
one thing that could be done to increase the certainty here would be to run tests against Sauce Labs in addition to the travis-ci/phantomJS build that happens now.  Sauce Labs will run tests on any browser  – and it's free for open source projects like this one. https://saucelabs.com/opensauce